timeless/reservations/views.py

```diff
@@ -97,6 +97,14 @@ def get(self, company_id):
 
 @bp.route("/")
 def list_reservations(reservations):
+
+    reservations = Reservation.objects.all()
+
+    reservations = {'reservations': reservations}
+
+#    	return render_template(
```

Comment:
@paulodamaso Why is here commented code, is it useless?

Reply:
@vladarefiev I'm sorry, I've commited the code of another issue